samples/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,44 @@
+# Microsoft XmlSerializer Generator
+
+Provides SGen tool to generate Xml Serialization code for types in specified assembly in order to improve the startup performance of a XmlSerializer when it serializes or deserializes objects of the specified types.
+
+## Prequisition:
+
+* [Install .NET Core 2.1 SDK](https://github.com/dotnet/core-setup#daily-builds) into a supported developer configuration.
```

Comment:
`.NET Core 2.1 SDK` -> `.NET Core SDK 2.1.0 preview daily builds`

The link should be https://github.com/dotnet/cli#installers-and-binaries

We may also want to include the version of the build that we've verified.